packages/critters-webpack-plugin/test/__snapshots__/index.test.js.snap

```diff
@@ -161,6 +161,39 @@ exports[`Inline <style> pruning should remove unused rules 1`] = `
 </body></html>"
 `;
 
+exports[`options { additionalStylesheets:["*.css"] } should match snapshot 1`] = `
+"<!DOCTYPE html><html><head>
+    <title>Additional Stylesheet CSS Demo</title>
+  <style>html {
+  padding: 0px;
+}
+
+html {
+  padding: 0px;
+}
```

Comment:
Unfortunately this is correct at the moment, and Critters no longer dedupes, see #88